index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <img src="images/image_geometry_1-1.png" alt="logo" class="shapes" />
+        <img src="images/ic_menu.png" alt="menu-bar" class="menu" />
+      </header>
+      <br />
+      <section>
+        <h1 class="intro">Emmanuella <br />Adu</h1>
+        <br />
+        <article class="about-me">
+          <p>
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Take a look at my works. If you like what you
+            see and have a project you need coded, do not hesitate and contact
+            me.
+          </p>
```

Comment:
- [ ] And Kindly make sure for your texts that you are using the same `fonts` as the ones in the template.😊